### PR TITLE
Washing machines

### PR DESCRIPTION
**Pull request by TDcoolguy300**
Replaces the sleep(200) with a spawn(200)

Throwing a wrench into a washing machine causes it to shake and flash red, eventually shooting it's contents out. 

http://puu.sh/oCI51/fb42600ea1.gif

### CODE REVIEW

**Comment by Crystalwarrior** [comment]
Based on http://decompressed-comic.tumblr.com/post/142767069488/the-timer-on-the-machine-is-now-complete-the

**Comment by ghost** [comment]
https://www.youtube.com/watch?v=vROdVsU_K80
Yes please lmao

File: code/game/machinery/washing_machine.dm [hunk 1]
```diff
@@ -37,65 +38,78 @@
...
+			qdel(crayon)
+			crayon = null
+
+		if(locate(/obj/item/weapon/wrench) in contents)
+			state = 1
+			for(var/atom/movable/O in contents)
+				O.loc = get_turf(src)
+				O.throw_at(get_edge_target_turf(src,pick(NORTH, SOUTH, EAST, WEST)), 500, 9, zone="chest")
+			update_icon()
+			working = 0
```
Comment (JohnGinnane) [inline code/game/machinery/washing_machine.dm:102]: Redundant as this is set in line 112

**Comment by JohnGinnane** [comment]
Code looks good!

File: code/game/machinery/washing_machine.dm (hunk 1 above)
Comment (TDcoolguy300) [inline code/game/machinery/washing_machine.dm:102]: No it's not, there's a return three lines down from this to prevent terrible bugs.

Comment (JohnGinnane) [inline code/game/machinery/washing_machine.dm:102]: Ah, I see that now, whoops!

**Comment by doobiedoo23** [comment]
i'm still getting fucking emails please remove my merge rights

**Comment by Crystalwarrior** [comment]
unsubscribe faggot
fag

**Comment by dreamcrab** [comment]
yes, do eet

**Comment by Nebyab** [comment]
yeah syre

**Comment by Crystalwarrior** [comment]
Doublehead bonanza, merging

**Pull request merged**